pkg/apis/core/types_shoot.go

```diff
@@ -66,6 +66,8 @@ type ShootSpec struct {
 	Addons *Addons
 	// CloudProfileName is a name of a CloudProfile object. This field is immutable.
 	CloudProfileName string
+	// CloudProfile is a reference to a CloudProfile or a NamespacedCloudProfile.
+	CloudProfile *CloudProfileReference
```

Comment:
Update this field in example shoot in `example/90-shoot.yaml`.

Reply:
The `CloudProfile` field is not supposed to be used as of now. When usage is sensible, the example will be added.